### ISSUE TITLE
High validation loss when fine-tuning Shakespeare on gpt-xl?

### ISSUE THREAD

**Issue by tombenj** [karpathy/nanoGPT#97]
It looks like the validation loss is significantly higher than the training loss. What in general is a good ratio when fine-tuning? perhaps changing the % of train vs validation instead of the current 90%?

```
iter 791: loss 1.2052, time 745.63ms
iter 792: loss 1.1750, time 745.47ms
iter 793: loss 1.0942, time 745.50ms
iter 794: loss 1.9608, time 745.64ms
iter 795: loss 0.6159, time 745.74ms
iter 796: loss 0.9267, time 745.76ms
iter 797: loss 2.4209, time 745.26ms
iter 798: loss 1.0178, time 746.26ms
iter 799: loss 0.6466, time 745.67ms
step 800: train loss 0.9795, val loss 3.7535
saving checkpoint to out-shakespeare
...
iter 997: loss 1.1173, time 747.40ms
iter 998: loss 0.2788, time 745.33ms
iter 999: loss 0.3390, time 745.36ms
step 1000: train loss 0.6703, val loss 3.8986
```

Samples look ok-ish:
```
ABHORSON:
Sir, you mistake.

SAMPSON:
Not seeing and studying are my necessities and means,
But acting what I see and understand.

GEORGE:
Master, they seek vengeance and none will have it.

ABHORSON:
Sir, they seek to revenge it and none will yield.

GEORGE:
They bargain for that end which no one can have.

ABHORSON:
Come, Master George,--

GEORGE:
Able, brave Abhorson, let me say it;
Let me begone.
I am not fit to set down my ignorance in
contrary to my education, nor to controvert
my false education by my true education's rewards.
Let me but be taught what I have seen, and so
I will serve.

ABHORSON:
Master, the king is coming to crave your services.

SAMPSON:
Awaketh he the king? no, to seek his Daniels.
```
